resotocore/resotocore/core_config.py

```diff
@@ -622,13 +625,31 @@ def parse_config(args: Namespace, core_config: Json, command_templates: Optional
     for key, value in args.config_override:
         set_from_cmd_line[ResotoCoreRootRE.sub("", key, 1)] = value
 
+    # json with all merged overrides for all components such as resotocore, resotoworker, etc.
+    all_config_overrides: Optional[Json] = None
+    # merge all provided overrides into a single object, preferring the values from the last override
+    for path in args.config_override_path:
+        with path.open() as f:
+            raw_yaml = yaml.safe_load(f)
+            merged = cast(Json, merge_json_elements(all_config_overrides or {}, raw_yaml))
+            all_config_overrides = merged
+
     # set the relevant value in the json config model
     migrated = migrate_core_config(core_config)
     adjusted = migrated.get(ResotoCoreRoot) or {}
     for path, value in set_from_cmd_line.items():
         if value is not None:
             adjusted = set_value_in_path(value, path, adjusted)
 
+    # merge the file overrides into the adjusted config
+    if all_config_overrides:
+        # here we only care about the resotocore overrides
+        core_overrides = all_config_overrides.get("resotocore", {})
```

Comment:
why is resotocore special here?

Reply:
that's the core only part of the config.